src/dialog/overflow-menu/overflow-menu.stories.ts

```diff
@@ -181,9 +150,24 @@ const CustomTemplate: Story<OverflowMenu> = (args) => ({
 export const Custom = CustomTemplate.bind({});
 Custom.storyName = "Custom Template";
 
-const DocumentationTemplate: Story = () => ({
+const CustomTriggerTemplate: Story<OverflowMenu> = (args) => ({
+	props: args,
 	template: `
-		<ibm-documentation src="documentation/modules/src_dialog_overflow_menu.html"></ibm-documentation>
+		<span>Overflow menu with custom trigger icon</span>
+		<ibm-overflow-menu
+			[flip]="flip"
+			[open]="open"
+			[customTrigger]="customTrigger"
+			[placement]="placement"
+			[offset]="offset">
+			<ibm-overflow-menu-option (selected)="selected($event)" (click)="click($event)">Option 1</ibm-overflow-menu-option>
+			<ibm-overflow-menu-option (selected)="selected($event)">Option 2</ibm-overflow-menu-option>
+			<ibm-overflow-menu-option disabled="true" (selected)="selected($event)">Disabled</ibm-overflow-menu-option>
+			<ibm-overflow-menu-option type="danger" (selected)="selected($event)">Danger option</ibm-overflow-menu-option>
+		</ibm-overflow-menu>
+		<ibm-placeholder></ibm-placeholder>
+		<ng-template #customTrigger><svg ibmIcon="document" size="16"></svg></ng-template>
 	`
 });
-export const Documentation = DocumentationTemplate.bind({});
+export const CustomTrigger = CustomTriggerTemplate.bind({});
+CustomTrigger.storyName = "With Custom Trigger";
```

Comment:
```suggestion
CustomTrigger.storyName = "With custom trigger";
```

Sentence capitalization?